### PR TITLE
Add Fantasia Smart Chain Mainnet

### PR DESCRIPTION
**Pull request by henry160186**

### CODE REVIEW

**Comment by henry160186** [comment]
Hi @ligi ,
Please approve my pull request. 
Thanks!

File: _data/chains/eip155-868.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "decimals": 18
+  },
+  "infoURL": "https://fantasia.technology/",
+  "shortName": "FSCMainnet",
+  "chainId": 868,
+  "networkId": 868,
+  "explorers": [
+    {
+      "name": "FSCScan",
+      "url": "https://explorer.fantasiachain.com",
```
Comment (ligi) [inline _data/chains/eip155-868.json:22]: not working

**Review by ligi — CHANGES_REQUESTED** [review]

File: _data/chains/eip155-868.json (hunk 1 above)
Comment (henry160186) [reply _data/chains/eip155-868.json:22]: It's fixing and re-deploy soon.
Can I submit without explorer and update later?

Comment (ligi) [reply _data/chains/eip155-868.json:22]: yea - RPCs same - add status incubating

**Comment by Eikyarkway** [comment]
<50

**Comment by Bladez1753** [comment]
From 999d69e6787747b21efc16fa5f4504285dff6b96 Mon Sep 17 00:00:00 2001
From: henry160186 <118771370+henry160186@users.noreply.github.com>
Date: Tue, 22 Nov 2022 18:13:34 +0700
Subject: [PATCH] Add Fantasia Smart Chain Mainnet

---
 _data/chains/eip155-868.json | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)
 create mode 100644 _data/chains/eip155-868.json

diff --git a/_data/chains/eip155-868.json b/_data/chains/eip155-868.json
new file mode 100644
index 0000000000..85dd7ec013
--- /dev/null
+++ b/_data/chains/eip155-868.json
@@ -0,0 +1,26 @@
+{
+  "name": "Fantasia Chain Mainnet",
+  "chain": "FSC",
+  "rpc": [
+    "https://mainnet-data1.fantasiachain.com/",
+    "https://mainnet-data2.fantasiachain.com/",
+    "https://mainnet-data3.fantasiachain.com/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "FST",
+    "symbol": "FST",
+    "decimals": 18
+  },
+  "infoURL": "https://fantasia.technology/",
+  "shortName": "FSCMainnet",
+  "chainId": 868,
+  "networkId": 868,
+  "explorers": [
+    {
+      "name": "FSCScan",
+      "url": "https://explorer.fantasiachain.com",
+      "standard": "EIP3091"
+    }
+  ]
+}

**Comment by Bladez1753** [comment]
From 999d69e6787747b21efc16fa5f4504285dff6b96 Mon Sep 17 00:00:00 2001
From: henry160186 <118771370+henry160186@users.noreply.github.com>
Date: Tue, 22 Nov 2022 18:13:34 +0700
Subject: [PATCH] Add Fantasia Smart Chain Mainnet

---
 _data/chains/eip155-868.json | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)
 create mode 100644 _data/chains/eip155-868.json

diff --git a/_data/chains/eip155-868.json b/_data/chains/eip155-868.json
new file mode 100644
index 0000000000..85dd7ec013
--- /dev/null
+++ b/_data/chains/eip155-868.json
@@ -0,0 +1,26 @@
+{
+  "name": "Fantasia Chain Mainnet",
+  "chain": "FSC",
+  "rpc": [
+    "https://mainnet-data1.fantasiachain.com/",
+    "https://mainnet-data2.fantasiachain.com/",
+    "https://mainnet-data3.fantasiachain.com/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "FST",
+    "symbol": "FST",
+    "decimals": 18
+  },
+  "infoURL": "https://fantasia.technology/",
+  "shortName": "FSCMainnet",
+  "chainId": 868,
+  "networkId": 868,
+  "explorers": [
+    {
+      "name": "FSCScan",
+      "url": "https://explorer.fantasiachain.com",
+      "standard": "EIP3091"
+    }
+  ]
+}

**Review by Bladez1753 — APPROVED** [review]
From 999d69e6787747b21efc16fa5f4504285dff6b96 Mon Sep 17 00:00:00 2001
From: henry160186 <118771370+henry160186@users.noreply.github.com>
Date: Tue, 22 Nov 2022 18:13:34 +0700
Subject: [PATCH] Add Fantasia Smart Chain Mainnet

---
 _data/chains/eip155-868.json | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)
 create mode 100644 _data/chains/eip155-868.json

diff --git a/_data/chains/eip155-868.json b/_data/chains/eip155-868.json
new file mode 100644
index 0000000000..85dd7ec013
--- /dev/null
+++ b/_data/chains/eip155-868.json
@@ -0,0 +1,26 @@
+{
+  "name": "Fantasia Chain Mainnet",
+  "chain": "FSC",
+  "rpc": [
+    "https://mainnet-data1.fantasiachain.com/",
+    "https://mainnet-data2.fantasiachain.com/",
+    "https://mainnet-data3.fantasiachain.com/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "FST",
+    "symbol": "FST",
+    "decimals": 18
+  },
+  "infoURL": "https://fantasia.technology/",
+  "shortName": "FSCMainnet",
+  "chainId": 868,
+  "networkId": 868,
+  "explorers": [
+    {
+      "name": "FSCScan",
+      "url": "https://explorer.fantasiachain.com",
+      "standard": "EIP3091"
+    }
+  ]
+}118771370+henry160186@users.noreply.github.com

File: _data/chains/eip155-868.json (hunk 1 above)
Comment (henry160186) [reply _data/chains/eip155-868.json:22]: Hi @ligi ,
We resolved all functions: RPCs and explorer.
Please recheck and approve our request.
Thanks!

**Review by ligi — APPROVED** [review]

**Pull request merged**